faunadb/strconv_test.go

```diff
@@ -33,7 +33,7 @@ func TestStringifyFaunaValues(t *testing.T) {
 
 	assertString(t,
 		SetRefV{Parameters: map[string]Value{"x": StringV("y")}},
-		`SetRefV{ Parameters: map[string]Value{"x": "y"} }`,
+		`SetRefV{ Parameters: Obj{"x": "y"} }`,
```

Comment:
Same here. The rendered string is incorrect in relation to the `SetRefV` struct.

Reply:
I'll revert this back to using map[string]Value or ObjectV. However, the rendered string will still be incorrect. In this case it should render as `SetRefV{ Parameters: ObjectV{"x": StringV("y")} }`

Reply:
My latest feedback also applies here.